live-examples/css-examples/transforms/meta.json

```
@@ -1,23 +1,29 @@
 {
-    "pages": {
-        "rotate": {
-            "exampleCode": "./live-examples/css-examples/transforms/function-rotate.html",
-            "fileName": "function-rotate.html",
-            "title": "Демо CSS: rotate()",
-            "type": "css"
-        },
-        "transform": {
-            "exampleCode": "./live-examples/css-examples/transforms/transform.html",
-            "fileName": "transform.html",
-            "title": "Демо CSS: transform",
-            "type": "css"
-        },
-        "translate": {
-            "cssExampleSrc": "./live-examples/css-examples/transforms/translate.css",
-            "exampleCode": "./live-examples/css-examples/transforms/function-translate.html",
-            "fileName": "function-translate.html",
-            "title": "Демо CSS: translate()",
-            "type": "css"
-        }
+  "pages": {
+    "rotate": {
+      "exampleCode": "./live-examples/css-examples/transforms/function-rotate.html",
+      "fileName": "function-rotate.html",
+      "title": "Демо CSS: rotate()",
+      "type": "css"
+    },
+    "transform": {
+      "exampleCode": "./live-examples/css-examples/transforms/transform.html",
+      "fileName": "transform.html",
+      "title": "Демо CSS: transform",
+      "type": "css"
+    },
+    "translate": {
+      "cssExampleSrc": "./live-examples/css-examples/transforms/translate.css",
+      "exampleCode": "./live-examples/css-examples/transforms/function-translate.html",
+      "fileName": "function-translate.html",
+      "title": "Демо CSS: translate()",
+      "type": "css"
+    },
+    "functionScale": {
+      "exampleCode": "./live-examples/css-examples/transforms/function-scale.html",
+      "fileName": "function-scale.html",
+      "title": "CSS Демо: scale()",
```

```suggestion
      "title": "Демо CSS: scale()",
```
Хай буде як в інших записів тут